phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
No, just keep what you have. It's a separate matter but our .hidden class should probably have an !important attribute for these kinds of situations.